### PR TITLE
bench: use html tables when available (coq/coq#17559)

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by Zimmi48** [comment]
I can do that.

**Comment by Zimmi48** [comment]
In progress: https://github.com/Zimmi48/bot/actions/runs/4863072936

**Comment by SkySkimmer** [comment]
>In progress: Zimmi48/bot/actions/runs/4863072936

job finished, should I rerun the bench or do you know how to resend the hook?

**Comment by Zimmi48** [comment]
I've resent the hook.

**Comment by Zimmi48** [comment]
Ah sorry, you had re-run the job already.

File: src/actions.ml [hunk 1]
```diff
@@ -293,6 +293,7 @@ end
 let fetch_bench_results ~job_info () =
   let open BenchResults in
   let open Lwt.Syntax in
+  let code_wrap str = f "```\n%s\n```" str in
```
[the remark this answers is not in the document]
Comment (SkySkimmer) [reply src/actions.ml]: ok

**Review by JasonGross — APPROVED** [review]
This is nice.  @SkySkimmer Is this ready for merge?

**Comment by SkySkimmer** [comment]
yes

**Pull request merged**